config_test.go

```diff
@@ -66,7 +66,7 @@ func TestRawConfig(t *testing.T) {
 	assert.Equal(c.Facility, fac)
 	assert.NotEqual(c.Hostname, "")
 	assert.Equal(c.Poll, false)
-	assert.Equal(c.RootCAs, papertrail.RootCA())
+	assert.Equal(c.RootCAs.Subjects(), papertrail.RootCA().Subjects())
```

Comment:
See https://github.com/golang/go/issues/45891 and https://github.com/golang/go/issues/46057